daniel-bernardo/index.html

```diff
@@ -0,0 +1,240 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="alternate stylesheet" href="css/tema1/style.css" title="spring">
```

Comment:
The default CSS should not have `"alternate stylesheet"` as a value in rel attribute